lib/core/test/unit/Cardano/Wallet/Primitive/AddressDiscovery/SequentialSpec.hs

```diff
@@ -160,35 +149,6 @@ spec = do
     parallel $ describe "DerivationPrefix" $ do
         textRoundtrip (Proxy @DerivationPrefix)
 
-    let styles =
-            [ Style (Proxy @'UtxoExternal)
-            , Style (Proxy @'UtxoInternal)
-            ]
-
-    parallel $ describe "AddressPool (Shelley)" $ do
```

Comment:
I am assuming all properties here, both for internal/external pools, and for both Shelley/Icarus keys are reproduced, ie. no property specified here is omitted?

Reply:
Yes, these property tests have all been superseded by the tests in `Cardano.Wallet.Address.PoolSpec`.

Thanks to the fact that the new `Pool` type is polymorphic in the `addr`, we no longer have to specialize to different key types or chain types — we get this as a free theorem: If the pool works for `generator = id`, it works for every other address type.